### PR TITLE
Geode 7765: jar updates for v1.13.0

### PR DESCRIPTION
**Pull request by metatype**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [x] Is your initial contribution a single, squashed commit?

- [x] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java [hunk 1]
```diff
@@ -223,7 +220,7 @@ public void run2() throws CacheException {
   }
 
   private static String printMap(Map<String, Object[]> m) {
-    Iterator<Map.Entry<String, Object[]>> itr = m.entrySet().iterator();
+    java.util.Iterator<java.util.Map.Entry<String, Object[]>> itr = m.entrySet().iterator();
```
Comment (kirklund) [inline geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java]: Why do Iterator and Map require fully qualified paths?

Comment (metatype) [reply geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java:223]: Good catch, something went wrong in auto-formatting.

**Comment by metatype** [comment]
Rolling back the jetty version, it's causing a test failure in `ListIndexManagementDUnitTest.getIndex_fails_when_region_name_is_missing_from_filter`.

**Comment by metatype** [comment]
Backed out mockito / powermock upgrades for a later day.  Newer versions of mockito generate a stack trace on every invocation and that causes excess runtime in a few concurrent tests.

**Comment by metatype** [comment]
The distributedTest failure is https://issues.apache.org/jira/browse/GEODE-6489.  @kirklund do you have any concerns about merging?

**Pull request merged**